gamemodes/terrortown/gamemode/client/cl_lang.lua

```diff
@@ -460,6 +496,31 @@ function LANG.ProcessMsg(name, params, mode)
 	LANG.ShowStyledMsg(text, LANG.GetStyle(name, mode))
 end
 
+---
+-- Returns the coverage in percent of the given language compared to the
+-- default language (english).
+-- @param string langName The name of the language in question
+-- @return number The coverage in percent
+-- @realm client
+function LANG.GetDefaultCoverage(langName)
+	-- if language is set to auto, get server default
+	if langName == "auto" then
+		langName = LANG.ServerLanguage
+	end
+
+	local englishTbl = LANG.Strings[LANG.DefaultLanguage]
+
+	return table.GetEqualEntriesAmount(LANG.Strings[langName], englishTbl) / table.Count(englishTbl)
```

Comment:
No rounding regarding floats / decimals?

Reply:
it is rounded in the UI. I thought it is ok if this function privides maximal accuracy and the user can decide if and how he wants to round it.